rustls/src/crypto/aws_lc_rs/sign.rs

```diff
@@ -347,6 +333,21 @@ impl SigningKey for Ed25519SigningKey {
     }
 }
 
+impl TryFrom<&PrivatePkcs8KeyDer<'_>> for Ed25519SigningKey {
+    type Error = Error;
+
+    /// Parse `der` as any EdDSA key type, returning the first which works.
+    ///
+    /// Note that, at the time of writing, Ed25519 does not have wide support
+    /// in browsers.  It is also not supported by the WebPKI, because the
+    /// CA/Browser Forum Baseline Requirements do not support it for publicly
+    /// trusted certificates.
+    fn try_from(der: &PrivatePkcs8KeyDer<'_>) -> Result<Self, Error> {
+        // TODO: Add support for Ed448
```

Comment:
I think this type name is badly named if the intention is it can support ed448. I suggest removing this comment and keeping the type name as-is (and also maybe fixing the second argument in its constructor?  But much less important given it is private.)